README.md

```diff
@@ -94,8 +94,12 @@ As we embrace true decentralization and permissionless platforms on Gnoland, we
      - `symbol`: The abbreviation your token's name, AKA the ticker. Please capitalize all letters.
      - `decimals`: The decimals of your token.
      - `chain_id`: The chain ID of your token.
-     - `description`: A description of your token. You can write up to 1500 letters.
+     - `description`: A description of your token. You can write up to 70 letters.
+     - `description_details`: A description details of your token. You can write up to 1500 letters.
```

Comment:
decsription should be 1500, decription_details should be 70 letters, right?

Reply:
After discussion Removed the description_details data column !